reminders/tests.py

```diff
@@ -72,13 +82,32 @@ def test_exist_reminder(self):
 
     def test_reminder_with_invalid_time(self, participant0, date_time=TIME_0):  # noqa: F811
         date_time = datetime(2022, 3, 22, 12, 12, 12, 0, tzinfo=timezone.utc)
-        expected = f'{time_format(date_time)} should be bigger than current.'
 
-        with pytest.raises(ValidationError, match=expected):
+        with pytest.raises(IntegrityError, match=PAST_DATE_TIME_ERROR):
             save_reminder(
                 create_reminder(participant0, METHOD_0, MESSAGES_0, date_time)
```

Comment:
done, date_time removed from the function parameters,
but because I don't want to add another global parameter I have decided to make the date_time a variable inside that specific function only, 
The date time variable I have entered is not like TIME_0 and needs to cause an exception so the test would work.